aspnetcore/blazor/hosting-models.md

```diff
@@ -217,7 +268,18 @@ Blazor Server apps have complete .NET API compatibility, while Blazor WebAssembl
 
 ### Direct access to server and network resources
 
-:::moniker range=">= aspnetcore-6.0"
+:::moniker range=">= aspnetcore-8.0"
+
+Components rendered for the Blazor Server hosting model have direct access to server and network resources where the app is executing. Because components rendered for the Blazor WebAssembly hosting model, static Blazor WebAssembly apps, and Blazor Hybrid apps execute on a client, they don't have direct access to server and network resources. Components can access server and network resources *indirectly* via protected server-based APIs. Server-based APIs might be available via third-party libraries, packages, and services. Take into account the following considerations:
```

Comment:
```suggestion
Components rendered for the Blazor Server hosting model have direct access to server and network resources where the app is executing. Because components hosted using Blazor WebAssembly or Blazor Hybrid execute on a client, they don't have direct access to server and network resources. Components can access server and network resources *indirectly* via protected server-based APIs. Server-based APIs might be available via third-party libraries, packages, and services. Take into account the following considerations:
```